experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -246,6 +246,11 @@ const listPrototypeProperties: PropertyDescriptorMap = {
 	[Symbol.iterator]: {
 		value: Array.prototype[Symbol.iterator],
 	},
+	at: {
+		value(this: SharedTreeList<AllowedTypes, "javaScript">, index: number): void {
+			getSequenceField(this).at(index);
+		},
+	},
```

Comment:
(new version of this comment)
@DLehenbauer I'm still trying to wrap my mind around this file and what exactly it's doing, so unsure about this chang and wanted to double check. The fact that LazySequence already had an at() method through LazyField but this wasn't here makes me particularly unsure if it's correct.

Reply:
You're doing the right thing.

The reason the code did not originally call LazySequence.at() or .map() [yet] was because I hadn't yet checked those implementations for strict compatibility (e.g., negative indices, etc.)